chia/server/server.py

```diff
@@ -599,29 +599,16 @@ async def send_to_specific(self, messages: List[Message], node_id: bytes32):
                 await connection.send_message(message)
 
     def get_outgoing_connections(self) -> List[WSChiaConnection]:
-        result = []
-        for _, connection in self.all_connections.items():
-            if connection.is_outbound:
-                result.append(connection)
-
-        return result
+        return [c for c in self.all_connections.values() if c.is_outbound]
 
     def get_full_node_outgoing_connections(self) -> List[WSChiaConnection]:
-        result = []
-        connections = self.get_full_node_connections()
-        for connection in connections:
-            if connection.is_outbound:
-                result.append(connection)
-        return result
+        return [c for c in self.get_full_node_connections() if c.is_outbound]
```

Comment:
Here and in places like https://github.com/Chia-Network/chia-blockchain/blob/108647112c7c5d3f2b05418f4eb35d99e0ec3958/chia/wallet/wallet_node.py#L576
little use is made of the fact that `get_full_node_connections` returns only after casting to a `list`.

Reply:
Maybe `conn` would be a bit more descriptive than raw `c`

Reply:
It is more descriptive, the question is whether there'a a chance that it doesn't become clear from context in
```python
    def get_full_node_outgoing_connections(self) -> List[WSChiaConnection]:
        return [c for c in self.get_full_node_connections() if c.is_outbound]
```
as
```python
        return [conn for conn in self.get_full_node_connections() if conn.is_outbound]
```
is more bulky.

Reply:
Right, I agree completely